modules/python/pylib/syslogng/__init__.py

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env python
+#############################################################################
```

Comment:
I don't think we need a shebang line here and neither of other modules in the same package.